test/e2e/l7networkpolicy_test.go

```diff
@@ -333,3 +347,63 @@ func testL7NetworkPolicyTLS(t *testing.T, data *TestData) {
 	probeL7NetworkPolicyTLS(t, data, clientPodName, "apis.google.com", false)
 	probeL7NetworkPolicyTLS(t, data, clientPodName, "www.facebook.com", true)
 }
+
+func testL7NetworkPolicyLogging(t *testing.T, data *TestData) {
+	l7LoggingNode := nodeName(0)
+	l7LogDir := logDir + "l7engine/eve-" + time.Now().Format(time.DateOnly) + ".json"
+
+	clientPodName := "test-l7-logging-client-selected"
+	clientPodLabels := map[string]string{"test-l7-logging-e2e": "client"}
+	cmd := []string{"bash", "-c", "sleep 3600"}
+	require.NoError(t, NewPodBuilder(clientPodName, data.testNamespace, agnhostImage).OnNode(l7LoggingNode).WithCommand(cmd).WithLabels(clientPodLabels).Create(data))
+	_, err := data.podWaitForIPs(defaultTimeout, clientPodName, data.testNamespace)
+	if err != nil {
+		t.Fatalf("Error when waiting for IP for Pod '%s': %v", clientPodName, err)
+	}
+	require.NoError(t, data.podWaitForRunning(defaultTimeout, clientPodName, data.testNamespace))
+
+	serverPodName := "test-l7-logging-server"
+	serverPodLabels := map[string]string{"test-l7-logging-e2e": "server"}
+	cmd = []string{"bash", "-c", "/agnhost netexec --http-port=8080"}
+	require.NoError(t, NewPodBuilder(serverPodName, data.testNamespace, agnhostImage).OnNode(l7LoggingNode).WithCommand(cmd).WithLabels(serverPodLabels).Create(data))
+	podIPs, err := data.podWaitForIPs(defaultTimeout, serverPodName, data.testNamespace)
+	if err != nil {
+		t.Fatalf("Error when waiting for IP for Pod '%s': %v", serverPodName, err)
+	}
+	require.NoError(t, data.podWaitForRunning(defaultTimeout, serverPodName, data.testNamespace))
+	serverIPs := getPodIPs(podIPs)
+
+	policyAllowPathHostname := "test-l7-http-allow-path-hostname"
+	l7ProtocolAllowsPathHostname := []crdv1beta1.L7Protocol{
+		{
+			HTTP: &crdv1beta1.HTTPProtocol{
+				Method: "GET",
+				Path:   "/host*",
+			},
+		},
+	}
+	// Create an L7 NetworkPolicies that allows HTTP path 'hostname', and probe twice
+	// where HTTP path 'hostname' is allowed yet 'clientip' will be rejected.
```

Comment:
```suggestion
	// Create an L7 NetworkPolicy that allows HTTP path 'hostname', and probe twice
	// where HTTP path 'hostname' is allowed yet 'clientip' will be rejected.
```

Reply:
Not sure which is correct, `an L7` or `a L7`. It depends on the pronunciation of `L7`.

Reply:
Changed to `one` to avoid confusion.